File: src/main/java/org/neo/gomina/model/sonar/SonarConfig.java
package org.neo.gomina.model.sonar;

import org.apache.commons.lang3.builder.ToStringBuilder;
import org.apache.commons.lang3.builder.ToStringStyle;

public class SonarConfig {

    public String mode;
    public String url;

    @Override
    public String toString() {
        return new ToStringBuilder(this, ToStringStyle.SHORT_PREFIX_STYLE)
                .append("mode", mode)
                .append("url", url)
                .toString();
    }
}

File: src/main/java/org/neo/gomina/integration/sonar/SonarConfig.kt
package org.neo.gomina.integration.sonar

import java.util.*

data class SonarServer(val id: String = "", val mode: String, val url: String = "")

data class SonarConfig (val servers: List<SonarServer> = ArrayList()) {
    val serverMap = servers.associateBy { it.id }
}